vividus-plugin-ssh/src/main/java/org/vividus/bdd/steps/ssh/SshSteps.java

```diff
@@ -109,14 +110,35 @@ public SftpOutput saveSftpResult(Commands commands, String server, Set<VariableS
      */
     @When("I create file with content `$content` at path `$destination` on $server over SFTP")
     public void createFileOverSftp(String content, String destination, String server) throws CommandExecutionException
+    {
+        executePutCommand(SftpCommand.PUT, List.of(content, destination), server);
+    }
+
+    /**
+     * Step retrieves server configuration by key, opens SFTP session and put local file at remote destination
+     * @param filePath    File path of file for putting on server
+     * @param destination Remote file destination
+     * @param server      Server key matching any of configured ones
+     * @throws IOException if I/O exception occurs during file reading
+     * @throws CommandExecutionException if any error happens during file creation
+     */
+    @When("I copy local file located at `$filePath` to path `$destination` on $server over SFTP")
+    public void copyFileOverSftp(String filePath, String destination, String server)
```

Comment:
could it be `Path filePath` ?

Reply:
Same as 'File file' - in this case SingleCommand, SftpCommand parameters and private fields tupes  will become more complicated. Do you think it's worth doing here?